lib/open_mic.rb

```diff
@@ -0,0 +1,24 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(attributes)
+    @location = attributes[:location]
+    @date = attributes[:date]
+    @performers = []
```

Comment:
great job noticing that this class takes a hash as an argument, and using correct bracket notation to access the information